### ISSUE TITLE
Provide a .desktop file

### ISSUE THREAD

**Issue by maieul** [TeX-Live/tlcockpit#6]
For Gnome / KDE and other desktop manager, it would be good to provide a launcher with a .desktop file

https://developer.gnome.org/integration-guide/stable/desktop-files.html.en

**Comment by norbusan** [comment]
In principle this is not a problem, but it wouldn't have any consequence to ship a .desktop file in TeX Live as it will not be installed into the required directories (/usr{,/local}/share/applications...). What is your suggested use of the .desktop file?

**Comment by maieul** [comment]
Yes, it is a problem. But as TeXLiveCocpit is is Gui, it should be available with "normal" user interface, that is graphic.

**Comment by norbusan** [comment]
Well said, but this will either need an independent installation routine for tlcockpit (need to think about that) or packaging by distributors.

**Comment by maieul** [comment]
yes, it could be an option at the end of texlive installer?

**Comment by norbusan** [comment]
At TeX Live we are not overly eager to set up integration in the installer, since this is hard to get right for all the supported OSs and architectures, not to speak of different distributions on these arch/os combinations. The .desktop is a good standard - but mostly relevant only to linux and not on some other systems.

So coming up with a reasonable system integration routine is nearly impossible, and the outcome very error-prone.
